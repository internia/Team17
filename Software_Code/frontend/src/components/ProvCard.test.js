// import React from 'react';
// import { render } from '@testing-library/react';
// import ProvCard from './ProvCard';
// import ReactDOM from 'react-dom';
// import { configure, shallow } from 'enzyme';
// import Adapter from 'enzyme-adapter-react-16';
// import { isCompositeComponent } from 'react-dom/test-utils';
// import renderer from 'react-test-renderer';

// configure({ adapter: new Adapter() })


// it('renders without crashing', () => {
//     const div = document.createElement("div");
//     ReactDOM.render(<ProvCard></ProvCard>, div)
// })

// it('renders searchbar correctly', () => {
//     function callBack() {
//         console.log("called back")
//     }
//     render(<ProvCard></ProvCard>)
// })


// it("matches snapshot", () => {
//     function callBack() {
//         console.log("called back")
//     }
//     const tree = renderer.create(<ProvCard></ProvCard>)
//     expect(tree).toMatchSnapshot();
//     // expect(true).toBeTruthy();
// })

it("TODO add propper props to test", ()=>{
    expect(true).toBeTruthy();
})