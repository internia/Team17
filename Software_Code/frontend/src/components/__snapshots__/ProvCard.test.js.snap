// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="sc-AykKC eKFqan"
>
  <div
    className="row align-items-center"
    style={
      Object {
        "backgroundColor": "#0b9cdb26",
        "margin-top": "1em",
      }
    }
  >
    <div
      className="col-12"
      style={
        Object {
          "padding-top": "1em",
        }
      }
    >
      <p
        style={
          Object {
            "fontSize": "1em",
          }
        }
      >
        <b>
          test
        </b>
      </p>
      <p
        style={
          Object {
            "fontSize": "0.8em",
          }
        }
      >
        , 
      </p>
    </div>
  </div>
</div>
`;
